internal/testutil/totp/totp_test.go

```diff
@@ -32,6 +35,8 @@ func TestGenerateOTPCode(t *testing.T) {
 	}
 
 	for _, test := range tests {
+		// This line can be removed when we upgrade golangci-lint to 1.58.1 and go1.22+
```

Comment:
It seems unlikely that we will remember to remove this comment when someday we delete hundreds of similar lines from the project. Maybe don't need it.

Reply:
Removed comment.